qlib/Mime.qm

```diff
@@ -403,6 +409,26 @@ public namespace Mime {
         "yaml": MimeTypeYaml,
         "zip": "application/zip",
         );
+
+    #! hash of non-alphanumeric characters that can be used unencoded in URL encoded format
+    /** @note the plus character can also appear unencoded in a URL string, but because many
+        decoders interpret it as a space (which is it in the query part of a URL string), it's
+        not in this list
+
+        @see <a href="https://tools.ietf.org/html/rfc1738">RFC 2738 2.2</a>
+    */
+    public const UrlEncodedChars = (
+        "\$": True,
+        "-": True,
+        "_": True,
+        ".": True,
+        "!": True,
+        "*": True,
+        "'": True,
+        "(": True,
+        ")": True,
+        ",": True
+        );
```

Comment:
According to https://tools.ietf.org/html/rfc3986#section-2.1, "~" (tilde) character should be here too.